### ISSUE TITLE
Add this to the Binder docs or the community forum?

### ISSUE THREAD

**Issue by choldgraf** [ian-r-rose/binder-workspace-demo#3]
In https://discourse.jupyter.org/t/open-multiple-notebooks-with-nbgitpuller/6690/2 somebody noted that the information in this repository was quite useful, though a bit difficult to discover. Think it'd be useful to upstream these docs somewhere, or write a short post in the forum so that it's more searchable?

**Comment by ian-r-rose** [comment]
Yes, I'd be happy for this to be more visible (either binder docs or JupyterLab, or both). We currently link it in the binder docs [here](https://mybinder.readthedocs.io/en/latest/howto/lab_workspaces.html?highlight=workspace#share-jupyterlab-workspaces-with-a-repository).

Part of the appeal of this repo, I think, is that it is a live binder that describes itself, so adding it to more static docs would lose that.

**Comment by choldgraf** [comment]
Anything that makes it more discoverable is +1 from me, we don't have to change the content 🙂
